iOS 13.4 beta 2 issue with GridItemPreferences

Building the code in iOS 13.4 crashes 100% of the time with the following message:

```
Undefined symbols for architecture x86_64:
  "property wrapper backing initializer of Grid.Grid.(itemsPreferences in _E80F9D962772A65A2700B0A688C6472F) : [Swift.AnyHashable : Grid.GridItemPreferences]", referenced from:
      Grid.Grid.init<A, B where A == SwiftUI.ForEach<A1, A1.Element.ID, B1>, A1: Swift.RandomAccessCollection, B1: SwiftUI.View, A1.Element: Swift.Identifiable>(_: A1, item: (A1.Element) -> B1) -> Grid.Grid<SwiftUI.ForEach<A1, A1.Element.ID, B1>> in Grid.o
      Grid.Grid.init<A, B, C where A == SwiftUI.ForEach<A1, B1, C1>, A1: Swift.RandomAccessCollection, B1: Swift.Hashable, C1: SwiftUI.View>(_: A1, id: Swift.KeyPath<A1.Element, B1>, item: (A1.Element) -> C1) -> Grid.Grid<SwiftUI.ForEach<A1, B1, C1>> in Grid.o
      Grid.Grid.init<A where A == SwiftUI.ForEach<Swift.Range<Swift.Int>, Swift.Int, A1>, A1: SwiftUI.View>(_: Swift.Range<Swift.Int>, item: (Swift.Int) -> A1) -> Grid.Grid<SwiftUI.ForEach<Swift.Range<Swift.Int>, Swift.Int, A1>> in Grid.o
      Grid.Grid.init<A, B where A == SwiftUI.TupleView<(A1, B1)>, A1: SwiftUI.View, B1: SwiftUI.View>(content: () -> SwiftUI.TupleView<(A1, B1)>) -> Grid.Grid<SwiftUI.TupleView<(A1, B1)>> in Grid.o
      Grid.Grid.init<A, B, C where A == SwiftUI.TupleView<(A1, B1, C1)>, A1: SwiftUI.View, B1: SwiftUI.View, C1: SwiftUI.View>(content: () -> SwiftUI.TupleView<(A1, B1, C1)>) -> Grid.Grid<SwiftUI.TupleView<(A1, B1, C1)>> in Grid.o
      Grid.Grid.init<A, B, C, D where A == SwiftUI.TupleView<(A1, B1, C1, D1)>, A1: SwiftUI.View, B1: SwiftUI.View, C1: SwiftUI.View, D1: SwiftUI.View>(content: () -> SwiftUI.TupleView<(A1, B1, C1, D1)>) -> Grid.Grid<SwiftUI.TupleView<(A1, B1, C1, D1)>> in Grid.o
      Grid.Grid.init<A, B, C, D, E where A == SwiftUI.TupleView<(A1, B1, C1, D1, E1)>, A1: SwiftUI.View, B1: SwiftUI.View, C1: SwiftUI.View, D1: SwiftUI.View, E1: SwiftUI.View>(content: () -> SwiftUI.TupleView<(A1, B1, C1, D1, E1)>) -> Grid.Grid<SwiftUI.TupleView<(A1, B1, C1, D1, E1)>> in Grid.o
      ...
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

```

If you remove the `@State` from `@State private var itemsPreferences` [here](https://github.com/spacenation/swiftui-grid/blob/512de567350d05037c937175107870f76d7e26a8/Sources/Grid/Grid.swift#L12)

and the whole `.onPreferenceChange` viewmodifier, so lines 17-26 [here](https://github.com/spacenation/swiftui-grid/blob/512de567350d05037c937175107870f76d7e26a8/Sources/Grid/Grid.swift#L17)

this issue goes away.

--- a/Sources/Grid/Grid.swift
+++ b/Sources/Grid/Grid.swift
@@ -1,15 +1,10 @@
 import SwiftUI
 
-struct GridItem: Identifiable {
-    let view: AnyView
-    let id: AnyHashable
-}
-
 /// A view that arranges its children in a grid.
 public struct Grid<Content>: View where Content: View {
     @Environment(\.gridStyle) private var style
+    @State var itemsPreferences: [AnyHashable : GridItemPreferences] = [:]
     let items: [GridItem]
-    @State private var itemsPreferences: [AnyHashable : GridItemPreferences] = [:]
     
     public var body: some View {
         GeometryReader { geometry in

--- a/Sources/Grid/GridItem.swift
+++ b/Sources/Grid/GridItem.swift
@@ -1,0 +1,6 @@
+import SwiftUI
+
+struct GridItem: Identifiable {
+    let view: AnyView
+    let id: AnyHashable
+}

